sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/reflect/DoFnInvokersTest.java

```diff
@@ -93,7 +93,8 @@
 @SuppressWarnings({
   "rawtypes", // TODO(https://github.com/apache/beam/issues/20447)
   "SameNameButDifferent",
-  "unused" // TODO(BEAM-13271): Remove when new version of errorprone is released (2.11.0)
+  "unused" // TODO(https://github.com/apache/beam/issues/21230): Remove when new version of
+  // errorprone is released (2.11.0)
```

Comment:
These look weird, any chance your automation could move these comments to the line above?

Reply:
I think this was actually caused by the autoformatter (spotlessApply), not my automation itself. With that said, I manually went through and cleaned it up (and some other weird formatting things)

Reply:
Thanks!